Mac-App/Mac-App/macOSMenu/macOSMenu.swift

```diff
@@ -10,7 +10,7 @@ import Cocoa
 import Frontend
 
 struct macOSMenu {
-    func triggerNewSearch(_ sender: NSMenuItem) {
+    func triggerNewSearch(_: NSMenuItem) {
```

Comment:
Can we remove _: NSMenuItem as well?

Reply:
Hmm you're right, it's not being used so we pass it. Will remove it 👍

Reply:
https://github.com/Thurman1776-/Uncomplect-us/pull/26/commits/c95c9af2b4406aef08f994610ebbbcaf182ef5af